packages/jest-config/src/normalize.js

```diff
@@ -354,6 +354,9 @@ function normalize(config: InitialConfig, argv: Object = {}) {
       case 'persistModuleRegistryBetweenSpecs':
       case 'preset':
       case 'replname':
+      case 'reporters':
+        value = _replaceRootDirTags(config.rootDir, config[key]);
```

Comment:
@thymikee So, If you take a look at the #2115 it adds in a type validation here, which is doing `Array.isArray` to type check if it's an array, but do we really need that here. As I am getting a ValidationError just based on what's there in validConfig.js. 

Here's the ValidationError, I think this error should be suffice. Do we need a custom error here.

```js
● Validation Error:

  Option "reporters" must be of type:
    array
  but instead received:
    string

  Example:
  {
    "reporters": [["./here-it-goes.js", {"option1": true}]]
  }

  Configuration Documentation:
  https://facebook.github.io/jest/docs/configuration.html


```